modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/TextStyles.kt

```diff
@@ -57,13 +57,16 @@ fun TextH20(
     modifier: Modifier = Modifier,
     color: Color = MaterialTheme.theme.colors.primaryText01,
     maxLines: Int = Int.MAX_VALUE,
-    textAlign: TextAlign? = null
+    textAlign: TextAlign? = null,
+    disableScale: Boolean = false,
+    fontSize: TextUnit = 22.sp,
+    lineHeight: TextUnit = 30.sp,
 ) {
     Text(
         text = text,
         color = color,
-        fontSize = 22.sp,
-        lineHeight = 30.sp,
+        fontSize = if (disableScale) fontSize.value.nonScaledSp else fontSize,
```

Comment:
One minor issue that I think this logic creates is that someone with a font scaled to 1.1 will have a bigger font than someone whose font is set to be 1.2 because the 1.2 gets reset to 1.0. _Very_ minor issue, but I wanted to mention it.